packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec

```diff
@@ -1,26 +1,30 @@
-# Generated from pulp_docker_client-4.0.0b6.dev01562331743.gem by gem2rpm -*- rpm-spec -*-
 # template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-%global gem_name pulp_docker_client
+%global gem_name pulp_container_client
+
+%global release 1
+%global prerelease rc1
+%global prereleaserpm %{?prerelease:.}%{?prerelease}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.0b6.dev01562331743
-Release: 1%{?dist}
-Summary: Pulp 3 Docker API Ruby Gem
+Version: 1.0.0
+
+Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
+Summary: Pulp container plugin for Pulp3 API Ruby Gem
 Group: Development/Languages
-License: GPLv2
-URL: https://rubygems.org/gems/pulp_docker_client
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+License: GPLv2+
+URL: https://github.com/pulp/pulp_container/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
+
+Obsoletes: %{?scl_prefix_ruby}rubygem(pulp_docker_client)
```

Comment:
Can you please use rubygem-name notation, not rubygem(name)? The latter freaks out rpm 4.15 (fedora 31)

Reply:
you mean just on this obsoletes line?  Or everywhere throughout the file?

Reply:
Just obsoletes